.github/workflows/hosted.yml

```diff
@@ -280,17 +281,22 @@ jobs:
       if: startswith(matrix.os, 'macos') && (matrix.target == 'cpp')
       run: echo "PATH=$(brew --prefix)/opt/ccache/libexec:$PATH" >> $GITHUB_ENV
 
-    - name: Build tool with Maven
+    - name: Build ANTLR tool testsuite with Maven
+      if: matrix.target == 'tool'
+      run: mvn install -DskipTests=true -Darguments="-Dmaven.javadoc.skip=true" -B -V
+
+    - name: Build ANTLR runtime testsuite with Maven
+      if: matrix.target != 'tool'
       run: mvn install -DskipTests=true -Darguments="-Dmaven.javadoc.skip=true" -B -V
```

Comment:
Thanks for noticing. Initially, I tried the following:

```
    - name: Build ANTLR tool testsuite with Maven
      if: matrix.target == 'tool'
      run: mvn install -pl tool-testsuite -DskipTests=true -Darguments="-Dmaven.javadoc.skip=true" -B -V

    - name: Build ANTLR runtime testsuite with Maven
      if: matrix.target != 'tool'
      run: mvn install -pl runtime-testsuite -DskipTests=true -Darguments="-Dmaven.javadoc.skip=true" -B -V
```

But it doesn't work because of weird errors related to caching. I reverted and forgot to get rid of redundancy.

Reply:
Ok, I think I'll leave it as is, since building a couple extra modules is not critical.